packages/thumbprint-react/components/Carousel/__snapshots__/test.tsx.snap

```diff
@@ -4,8 +4,6 @@ exports[`Carousel \`selectedIndex\` supports decimal \`selectedIndex\` values 1`
 <Carousel
   onSelectedIndexChange={[Function]}
   selectedIndex={2.3}
-  spacing="0px"
```

Comment:
I'd expect the typing to have had no effect on the the test snapshots. What changed here?

Reply:
this has happened with all of them. not sure if it's just a quirk of jest, but it seems like assigning default values via `defaultProps` vs in the function arguments affects whether they appear in the output. the default values all still work fine in real code though, so i'm assuming this is just a quirk of the snapshots.